6/체육복.js

```diff
@@ -0,0 +1,32 @@
+function solution(n, lost, reserve) {
+  let gymUniformCount = 0;
+  const students = [];
```

Comment:
```suggestion
  const students = [];
  let gymUniformCount = 0;
```
let은 재할당 될 수 있기 때문에 항상 const는 let위에 선언해줍니다!